# HG changeset patch
# User hodelas
# Date 959183619 0
#      Wed May 24 15:53:39 2000 +0000
# Node ID 9debe1be75a5072651489969d0458c54a7add941
# Parent  b0a68efa1295b5d95ad81dd2ad171bfe99fe9b1b
[project @ 2000-05-24 15:53:39 by hodelas]
Updated documentation; fixed typo, added warning about difference
with MATLAB dlqe function.

diff --git a/scripts/control/base/dlqe.m b/scripts/control/base/dlqe.m
--- a/scripts/control/base/dlqe.m
+++ b/scripts/control/base/dlqe.m
@@ -1,9 +1,9 @@
-## Copyright (C) 1993, 1994, 1995 Auburn University.  All rights reserved.
+## Copyright (C) 1993, 1994, 1995, 2000 Auburn University.  All rights reserved.
 ##
 ## This file is part of Octave.
 ##
 ## Octave is free software; you can redistribute it and/or modify it
 ## under the terms of the GNU General Public License as published by the
 ## Free Software Foundation; either version 2, or (at your option) any
 ## later version.
 ##
@@ -44,58 +44,59 @@
 ##
 ## If specified, @var{z} is @code{cov (@var{w}, @var{v})}.  Otherwise
 ## @code{cov (@var{w}, @var{v}) = 0}.
 ##
 ## The observer structure is
 ## @iftex
 ## @tex
 ## $$
-##  z_{k+1} = A z_k + B u_k + k (y_k - C z_k - D u_k)
+##  z_{k+1} = A z_k + B u_k + L (y_k - C z_k - D u_k)
 ## $$
 ## @end tex
 ## @end iftex
 ## @ifinfo
 ##
 ## @example
-## z[k+1] = A z[k] + B u[k] + k (y[k] - C z[k] - D u[k])
+## z[k+1] = A z[k] + B u[k] + L (y[k] - C z[k] - D u[k])
 ## @end example
 ## @end ifinfo
 ##
 ## @noindent
 ## The following values are returned:
 ##
 ## @table @var
 ## @item l
-## The observer gain,
+## The observer gain.  The estimator state matrix
 ## @iftex
 ## @tex
-## $(A - ALC)$.
+## $(A - LC)$
 ## @end tex
 ## @end iftex
 ## @ifinfo
-## (@var{a} - @var{a}@var{l}@var{c}).
+## (@var{a} - @var{l}@var{c})
 ## @end ifinfo
-## is stable.
+## is stable.   NOTE: This differs from the MATLAB dlqe function, which 
+## returns L such that (A - A L C) is stable.
 ##
 ## @item m
 ## The Riccati equation solution.
 ##
 ## @item p
 ## The estimate error covariance after the measurement update.
 ##
 ## @item e
 ## The closed loop poles of
 ## @iftex
 ## @tex
-## $(A - ALC)$.
+## $(A - LC)$.
 ## @end tex
 ## @end iftex
 ## @ifinfo
-## (@var{a} - @var{a}@var{l}@var{c}).
+## (@var{a} - @var{l}@var{c}).
 ## @end ifinfo
 ## @end table
 ## @end deftypefn
 
 ## Author: A. S. Hodel <a.s.hodel@eng.auburn.edu>
 ## Created: August 1993
 ## Modified for discrete time by R. Bruce Tenison (btenison@eng.auburn.edu)
 ## October, 1993
